### PR TITLE
Replace unused 0C bank with aglab's / scut's extended 0C bank 

### PR DESCRIPTION
**Pull request by Reonu**
This is used by a lot of music ports and it's overall a good feature for the repo to have.

The bank it's replacing is unused and useless. Any usecase that it might have is also covered by this bank.

### CODE REVIEW

File: sound/sound_banks/0C_extended_bank.json [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+        "Ele Guitar",
+        "Ele Piano",
+        "Grand Piano",
+        "French Horn",
+        "Pan Flute",
+        "Strings",
+        "Acoustic Guitar",
+        "Jingles",
+		"Hi-Hat",
+		"Crash Cymbal",
```
Comment (gheskett) [inline sound/sound_banks/0C_extended_bank.json:585]: These use two tabs, while everything else uses 8 spaces.

**Review by gheskett — CHANGES_REQUESTED** [review]

**Comment by gheskett** [comment]
Before approving, I will also look at the audio memory pools and make sure everything looks good. I'm currently out of town but I'll look at it when I have the opportunity.

**Review by gheskett — APPROVED** [review]
Extended bank should work fine in most use cases, based on what I've tested.

**Comment by arthurtilly** [comment]
looks good

**Review by thecozies — APPROVED** [review]

**Pull request merged**